styleguide/components/61-layouts/general/info-layout/_detail-layout.scss

```diff
@@ -0,0 +1,34 @@
+.detail-layout {
+  @extend %content-container;
+
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  margin-top: 1.2rem;
+  font-size: .9rem;
+  line-height: 1.8;
+
+  > .content {
+    width: span(8 of 12);
+    max-width: 36rem;
+    padding-right: span(1 of 12);
+
+    .image-gallery,
+    .video,
+    .quote,
+    .feedback-form {
+      @extend %wider;
```

Comment:
Reverse the logic
You've already got four components in the exception and only one component as default.

Reply:
I changed the logic.